app/views/catalog/_home_text.html.erb

```diff
@@ -1,9 +1,49 @@
-<h4><%= t('blacklight.welcome') %></h4>
-<p>&nbsp;</p>
-<p>To modify this text to your specifications, copy this file located in the blacklight plugin directory:<br/>
-&nbsp;&nbsp;&nbsp;<%= Blacklight.root %>/app/views/catalog/_home_text.html.erb</p>
-<p>to your top level rails app:<br/>
-&nbsp;&nbsp;&nbsp;<%= Rails.root %>/app/views/catalog/_home_text.html.erb</p>
-<p>Note that you will need to create the 'catalog' directory under views.</p>
-<% #Creates a bit space in the page -%>
+<div class="page-header row">
+  <h1 class="col-md-8"><%= t('blacklight.welcome') %></h1>
+  
+  <ul class="nav nav-pills col-md-4">
+    <li><a href="https://github.com/projectblacklight/blacklight/">Github</a></li>
+    <li><a href="https://github.com/projectblacklight/blacklight/wiki">Wiki</a></li>
+    <li><a href="https://github.com/projectblacklight/blacklight/releases/tag/v<%= Blacklight::VERSION %>">Release Notes</a></li>
+  </ul>
 
+
+</div>
+
+<div id="getting-started">
+  <h2>Here&rsquo;s how to get started:</h2>
+  
+  <ol>
+    <li>To modify this text, you need to <a href="http://guides.rubyonrails.org/engines.html#improving-engine-functionality">override the Blacklight-provided view</a>.
+You can copy this file, located in the blacklight gem: <br />
+<%= Blacklight.root %>/app/views/catalog/_home_text.html.erb <br />
```

Comment:
:+1: totally agree.

It's also not up to us to make sure folks don't put sensitive information in their file paths (even though I see it all the time...)  Just thought I would throw it out there since we're touching the file (for the first time since 2009 it looks like!)